libtextsecure/message_receiver.js

```diff
@@ -120,6 +114,28 @@ MessageReceiver.prototype.extend({
     //   all cached envelopes are processed.
     this.incoming = [this.pending];
   },
+  async startLocalServer() {
+    let myLokiIp;
+    let myServerPort;
```

Comment:
No need to declare those two outside the `try` block

Reply:
Oh try whoops